addons/product/models/product_pricelist_item.py

```diff
@@ -438,8 +493,8 @@ def _compute_base_price(self, product, quantity, uom, date, currency):
         return price
 
     def _compute_price_before_discount(self, *args, **kwargs):
-        """Compute the base price of the lowest pricelist rule whose pricelist discount_policy
-        is set to show the discount to the customer.
+        """Compute the base price of the lowest pricelist rule,
+        discount is shown by default if computation method is a percentage rule.
```

Comment:
Depending on dala's answer, we might want to drop this method & logic (only keep the last discount amount)